ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,23 +526,31 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
```

Comment:
What we want to do here is create a temporary variable like `var jobTdHtml` and start concatenating text to it. We'll only use a jQuery selector to add it to the DOM when we're done.